integration-tests/metaChecker.go

```diff
@@ -126,6 +126,16 @@ func checkMetaLogs(apiLog gjson.Result, logs map[string]*transaction.Log, txHash
 	return nil
 }
 
+func getIndexedLog(logs []*outport.LogData, txHash string) (*outport.LogData, bool) {
```

Comment:
maybe you can rename this function. 
1. getLogByTxHash
2. findLogByTxHash
3. searchLogByTxHash

Reply:
Right.
Renamed it to `getLogByTxHash`